examples/platform/silabs/SiWx917/SiWx917/sl_wifi_if.c

```diff
@@ -487,7 +476,7 @@ static void wfx_rsi_do_join(void)
         status = sl_net_set_credential(id, SL_NET_WIFI_PSK, &wfx_rsi.sec.passkey[0], strlen(wfx_rsi.sec.passkey));
         if (SL_STATUS_OK != status)
         {
-            return status;
+            return;
```

Comment:
I see this function is void and that is why you removed the status return. But I strongly suggest changing that and being able to return error codes. Currently, you can call `wfx_rsi_do_join` and have no idea if the function successfully joined or if it fails. That's problematic